code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -175,64 +174,48 @@
 	for(var/t in adjacent_turfs)
 		var/turf/open/enemy_tile = t
 
-		if(fire_count > enemy_tile.current_cycle)
-			enemy_tile.archive()
-
-		/******************* GROUP HANDLING START *****************************************************************/
-
-			var/should_share_air = FALSE
-			var/datum/gas_mixture/enemy_air = enemy_tile.air
-			if(enemy_tile.excited)
-				//cache for sanic speed
-				var/datum/excited_group/enemy_excited_group = enemy_tile.excited_group
-				if(our_excited_group)
-					if(enemy_excited_group)
-						if(our_excited_group != enemy_excited_group)
-							//combine groups (this also handles updating the excited_group var of all involved turfs)
-							our_excited_group.merge_groups(enemy_excited_group)
-							our_excited_group = excited_group //update our cache
-						should_share_air = TRUE
-					else
-						if((recently_active == 1 && enemy_tile.recently_active == 1) || our_air.compare(enemy_air))
```

Comment:
from what I can tell,  `(recently_active == 1 && enemy_tile.recently_active == 1) ==
 (our_excited_group && enemy_excited_group)`. there's no way for this to be false so these checks could never be true, making them useless